packages/expo-modules-core/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 
 ### 💡 Others
 
+- Logger changes. ([#18271](https://github.com/expo/expo/pull/18271) by [@douglowder](https://github.com/douglowder))
```

Comment:
Changelog entries must be descriptive (and concise), otherwise I don't see any reason to add it. Sometimes we're intentionally not adding any entries for non-user-facing changes, but I think `expo-modules-core` is kinda special package since its native code is an API for other 3rd-party libs, so it's better to add something in that case.
Maybe something like this:
```suggestion
- Changed access levels in the Logger and fixed the timer to log milliseconds instead of seconds. ([#18271](https://github.com/expo/expo/pull/18271) by [@douglowder](https://github.com/douglowder))
```